package.json

```diff
@@ -20,7 +21,8 @@
     "extends": "developit",
```

Comment:
Wouldn't it be better to remove the already present eslint config when we are using prettier?

Reply:
The eslint preset contains many linting rules not related to formatting. That's why I left it there 🙂

Reply:
All right, disregard my comment then :)